src/components/editor/PropertyComponent.jsx

```diff
@@ -33,7 +33,7 @@ export class PropertyComponent extends Component {
         break
       case "list":
         result = (<InputListLOC key = {this.props.index} reduxPath={[this.props.rtId, property.propertyURI]}
-                                propertyTemplate = {property} lookupConfig = {this.state.configuration}/>)
+                                propertyTemplate = {property} lookupConfig = {this.state.configuration[0]}/>)
```

Comment:
It would be ideal to add a test covering this change.

Reply:
Should this satisfy it? 
https://github.com/LD4P/sinopia_editor/blob/4c2925b7e2bb23c1b64303158d4a8977ec4963e7/__tests__/components/editor/PropertyComponent.test.js#L30

Reply:
I was thinking a typecheck test, yeah. What I had in mind was a test that made sure that a child `InputListLOC` component had a `lookupConfig` prop that looked how we expected it.